### PR TITLE
Update AndroidMobileCapabilityType

### PR DESCRIPTION
**Pull request by a-en**
## Change list

Added missing capabilities from [docs](http://appium.io/docs/en/writing-running-appium/caps/) to AndroidMobileCapabilityType interface. Also, this will fix [this issue](https://github.com/appium/java-client/issues/1236)
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [x] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted java code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java [hunk 1]
```diff
@@ -276,4 +373,54 @@
...
+    String UNLOCK_TYPE = "unlockType";
+
+    /**
+     * A key pattern to unlock used by {@code unlockType}.
+     */
+    String UNLOCK_KEY = "unlockKey";
+
+    /**
+     * Initializing the app under test automatically.
+     * Appium does not install/launch the app under test if this is {@code false}. Defaults to {@code true}
```
Comment (mykola-mokhnach) [inline src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java]: it does install the app, but skips the launch

**Comment by mykola-mokhnach** [comment]
Thanks. It might also be useful to mention since which Appium version these caps are available. (aka `@since` doc option)

File: src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java (hunk 1 above)
Comment (a-en) [reply src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java]: @mykola-mokhnach I've changed description to 'Appium does not launch the app...'. Should it be also changed in http://appium.io/docs/en/writing-running-appium/caps/ ?

**Comment by a-en** [comment]
@mykola-mokhnach  I've went through releases of Appium on github and collected '@since' for those caps I was able to find. Are there any other sources that might be used to add this information?

**Comment by mykola-mokhnach** [comment]
Usually version changelist is the only source to find it out

File: src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java (hunk 1 above)
Comment (mykola-mokhnach) [reply src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java]: yes, it makes sense to update the description in the documentation

**Pull request merged**